GoogleDataTransport/GDTCORLibrary/GDTCORStorage.m

```diff
@@ -309,15 +300,19 @@ + (BOOL)supportsSecureCoding {
 }
 
 - (instancetype)initWithCoder:(NSCoder *)aDecoder {
+  /**Sets a global translation mapping to decode GDTCORStoredEvent objects encoded as instances of
```

Comment:
These comments can begin with // instead of /**. Also, maintain a space between the comment delimiter and the beginning of your comment.